contracts/core/TribeRoles.sol

```diff
@@ -84,6 +84,12 @@ library TribeRoles {
     /// @notice capable of engaging with Votium for voting incentives.
     bytes32 internal constant VOTIUM_ROLE = keccak256("VOTIUM_ADMIN_ROLE");
 
+    /// @notice capable of adding an address to multi rate limited
+    bytes32 internal constant ADD_MINTER_ROLE = keccak256("ADD_MINTER_ROLE");
```

Comment:
Currently I believe OA can mint up to 100m Fei or something right?

Reply:
But yes fair point. We also may need to get this role integrated into the pods upgrade. Tagging @thomas-waite for his thoughts here.

Reply:
This permissions the adding of an address that can mint Fei? What address is being granted this role? What is the admin? Can't see in the fip script. 

This doesn't enable arbitrary minting - there is a rate limit - and it's likely to be quite operational in nature. So I'd lean towards making the `ROLE_ADMIN` the admin of this `ADD_MINTER_ROLE` so that the council can grant it out. 

As it is quite a powerful role, would want some level of lindy on the Council first

Reply:
Makes sense. We're not deploying the global rate limited minter in this pr, so we won't need to do any of the granting of the admin or anything in this pr either.